views/datetime/datecontrol.html

```diff
@@ -3,17 +3,18 @@
     <div class="input-group-prepend">
       <button class="btn btn-secondary flex-shrink-0" title="Previous {{dateControl.duration}}"
           ng-click="dateControl.shiftDate(-1)"
-          ng-disabled="dateControl.disabled"
-          ng-if="dateControl.duration !== 'custom'">
+          ng-disabled="dateControl.disabled">
         <i class="fa fa-chevron-left"></i>
       </button>
     </div>
-    <wheel-date title="Start date"
+    <wheel-date id="timelineStartDate"
```

Comment:
Prefer using a specific `class` selector instead of an `id`. Element ID's are expected to be unique within a page, but this can't be enforced. Instead we use a class that both implies it's used as a selector in JavaScript, and is descriptive of the directive it belongs to. For example, this might use `class="js-datecontrol__startDate"` and `class="js-datecontrol__endDate".

This combined with finding the selector from the directive's root element are a more reliable way to protect against finding the wrong element on the page.